app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/attachment/AttachmentOptions.kt

```diff
@@ -0,0 +1,78 @@
+package com.wire.android.ui.home.messagecomposer.attachment
+
+import android.util.Log
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.GridCells
+import androidx.compose.foundation.lazy.LazyVerticalGrid
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.wire.android.R
+import com.wire.android.ui.common.AttachmentButton
+
+@OptIn(
+    ExperimentalComposeUiApi::class,
+    ExperimentalFoundationApi::class
+)
+@Composable
+fun AttachmentOptionsComponent() {
+    val viewModel: AttachmentOptionsViewModel = hiltViewModel()
+    LazyVerticalGrid(
+        cells = GridCells.Fixed(4),
```

Comment:
I feel that this value can change depending on device type and screen orientation so maybe it should be moved to `wireDimensions` or even we should create another one `wireIntegers` as it's not actually a dimension 😅

Reply:
I agree on that one :P it is abit wishfull thinking in my opinion that it will looks correct on all screens ;p maybe there is a way to keep placing those buttons on a row untill it has space and then jump into the next line and keep placing those buttons if there are some left ? that way it will be more responsive but then you theoritacally could have a case on some small screen devices for example where you got 2 button on a row but have lets say 3 columns in total :P 

otherwise we could use some custom Layout measuring the screen width and dividing it by lets say 120.dp for each grid and then decide based on that how many rows in a column we want to use :P which kinds of does this I think : 

 cells = GridCells.Adaptive(minSize = 128.dp)